benchmarking/driver/benchmark_driver.py

```diff
@@ -25,6 +25,8 @@ def runOneBenchmark(info, benchmark, framework, platform,
     minfo = copy.deepcopy(info["treatment"])
     if "shared_libs" in info:
         minfo["shared_libs"] = info["shared_libs"]
+    if hasattr(getArgs(), "timeout") and "timeout" not in benchmark["model"]:
+        benchmark["model"]["timeout"] = getArgs().timeout
```

Comment:
I suggest to modify benchmark.py and update the meta field. So that the field value may be copied to "model" or "test".